recipes-wpe/wpeframework/wpeframework-ocdm-playready_git.bb

```diff
@@ -1,10 +0,0 @@
-require include/wpeframework-ocdm-playready.inc
```

Comment:
What is the reason for remove this recipe?

Reply:
We have two files for play ready 2.5 and 4.0 (wpeframework-ocdm-playready_2.5+git.bb & wpeframework-ocdm-playready_4.0+git.bb). So do we need to keep wpeframework-ocdm-playready_git.bb with same details of 4.0 ?